announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements.vm

```diff
@@ -371,7 +371,7 @@
 					#set ($rowCount=0)
 					<ul class="synopticList">
 					#foreach ($ann_item in $showMessagesList)
-						#set ($isHightlighted = $ann_item.getProperties().getProperty("highlight"))
+						#set ($isHightlighted = (true == $ann_item.getProperties().getProperty("highlight")))
```

Comment:
getProperty() returns a String?

so `true` should be quoted as a string?

if it works like this, it seems like Velocity just not being very strict?

Reply:
I borrowed the logic that's elsewhere in the file. If you do a search for "(true == $", you'll see other lines that do not quote true. Should these be changed as well?

Reply:
I'll force push a revision that places the 3 instances in this template where I've identified true should be a string.